kubernetes/setup-environment

```diff
@@ -11,8 +11,16 @@ done
 
 log "MariaDB container is ready"
 log "Setting up database"
-docker exec -i $(docker ps | grep velum-dashboard | awk '{print $1}') rake db:setup
-docker exec -e RAILS_ENV=test -i $(docker ps | grep velum-dashboard | awk '{print $1}') rake db:setup
+until docker exec -i $(docker ps | grep velum-dashboard | awk '{print $1}') rake db:setup;
+do
+    log "Waiting for mariadb to accept connections"
+    sleep 5
+done
+until docker exec -e RAILS_ENV=test -i $(docker ps | grep velum-dashboard | awk '{print $1}') rake db:setup;
```

Comment:
I think here we can just have this one statement, without the loop, as we just want to create the test database for the unit testing in the development environment. and by that time the mariadb connection should already be up accepting connections.